mlflow/deployments/constants.py

```diff
@@ -0,0 +1,20 @@
+from mlflow.environment_variables import _EnvironmentVariable
+
+# Specifies the timeout for deployment client APIs to declare a request has timed out
+MLFLOW_DEPLOYMENT_PREDICT_TIMEOUT = _EnvironmentVariable(
+    "MLFLOW_DEPLOYMENT_PREDICT_TIMEOUT", int, 120
+)
```

Comment:
Should we move this to `environment_variables.py`?

Reply:
I defined this here to avoid conflicts. After merging this PR, can you request a change to move this to `environment_variables.py`?